### ISSUE TITLE
Model fits the upper boddy well, but fails completly for lower body parts! Is this due to the wrong choice of the models?

### ISSUE THREAD

**Issue by eulerleibniz** [IDEA-Research/OSX#133]
I am testing your model with the following command:
```
python demo.py --gpu 0 --img_path input.jpg --output_folder output
```
I have the following output (Seems perfectly fine, just a warning):
> Using GPU: 0
/home/user_name/miniconda3/envs/myenv/lib/python3.8/site-packages/mmcv/__init__.py:20: UserWarning: On January 1, 2023, MMCV will release v2.0.0, in which it will remove components related to the training process and add a data transformation module. In addition, it will rename the package names mmcv to mmcv-lite and mmcv-full to mmcv. See https://github.com/open-mmlab/mmcv/blob/master/docs/en/compatibility.md for more details.
  warnings.warn(
06-13 04:22:40 Load checkpoint from ../pretrained_models/osx_l.pth.tar
06-13 04:22:40 Creating graph...
Load checkpoint from ../pretrained_models/osx_l.pth.tar
Using cache found in /home/user_name/.cache/torch/hub/ultralytics_yolov5_master
YOLOv5 🚀 2024-6-13 Python-3.8.19 torch-1.11.0+cu113 CUDA:0 (NVIDIA GeForce GTX 1070 Ti, 8111MiB)
Fusing layers... 
YOLOv5s summary: 213 layers, 7225885 parameters, 0 gradients
Adding AutoShape... 

Problem is that the **upper body** is well approximated but **lower body** is completely off. Specially in my single person images where the person is laying down or sitting with their feet offset from their body I get very awful results. (I can share images with you, but not doing it cause not sure of their copyright)

Am I using an incorrect model checkpoint? (shows `osx_l.pth.tar`). Am I using incorrect key-point detector (shows `YOLOV5`). Is your model trained on upper body only? How to change the models if there are options?
